openmc/summary.py

```diff
@@ -2,17 +2,22 @@
 
 import openmc
 
-try:
-    import h5py
-except ImportError:
-    msg = 'Unable to import h5py which is needed by openmc.summary'
-    raise ImportError(msg)
+
+_h5_imported = False
```

Comment:
Why is this needed? It would be cleaner to just re-import ``h5py`` each time a ``Summary`` is created (typically once), unless there is good reason to only import once.